tests/integration/conftest.py

```diff
@@ -32,9 +35,12 @@ def validate(request, export_test_cases_dir: Path):
         # Check if the validator is installed
         validator = get_validator()
         if validator is None:
-            pytest.fail("Run `cargo build -p release` to install the validator")
-    else:
-        pytest.skip("Skipping validation tests as requested")
+            if IN_CI:
+                pytest.fail("Validator not installed")
+            else:
+                pytest.skip(
+                    "Skipping validation: Run `cargo build` to install the validator"
```

Comment:
You can also just run `pytest --no-validation`, following #472. So this is a different take on the same idea. If we go with this, we might want to revert #472? Or, we might instead want to do something like that but for execution tests too?